install/setup_react.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env sh
+#
+# This script install the node dependencies listed in package.json and build the bundle file using webpack.
+
+# bring in the color variables: `normal`, `info`, `warning`, `danger`, `reset`
+. "$(dirname "$(readlink -f "$0")")/utils.sh"
+
+# Instead of using apt-get to install npm we will nvm to install npm because apt-get installs older-version of node
+echo "${normal}[*] Installing npm using nvm.${reset}"
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.1/install.sh | bash
+nvm install node
+nvm alias default node
```

Comment:
Don't use pipe to use on a script. This is a bad practice for an infosec tool :D